dump25/sql/claims_sql.py

```
@@ -0,0 +1,197 @@
+    # ---
+    create_tables()
+    # ---
+    dump_dir = Path(__file__).parent.parent / "parts"
+    # ---
+    files = [x for x in dump_dir.glob("*.json")]
+    # ---
+    nx = 0
+    # ---
+    for file in tqdm.tqdm(files):
+        # ---
+        nx += 1
```

Ruff (0.8.2)

Use `enumerate()` for index variable `nx` in `for` loop

(SIM113)